.travis.yml

```diff
@@ -19,7 +19,7 @@ install:
 - export PATH=$PATH:/tmp/godep/
 
 script:
-- make prerelease
+- make validate_commits prerelease
```

Comment:
-1, this is not going to work. Travis is setting up the get repo to be in a detached state. The current validation target is not going to work here.